numpy/lib/_function_base_impl.py

```diff
@@ -67,7 +67,7 @@
     # --- HYNDMAN and FAN METHODS
     # Discrete methods
     'inverted_cdf': {
-        'get_virtual_index': lambda n, quantiles: _inverted_cdf(n, quantiles),
+        'get_virtual_index': _inverted_cdf,
```

Comment:
the CI errors seem to suggest that `_inverted_cdf` isn't defined at this point

Reply:
Yes, it's defined later on. Is it OK to move the function definition around?

Reply:
Sure, but I wonder why I get this error for `_inverted_cdf` but not for other functions defined/used in similar places, such as `_closest_observation`.

Reply:
After fixing `_inverted_cdf`, I got the same error for `_closest_observation`. At least that's consistent! Fixed.